plugins/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -571,7 +571,7 @@ def delete_nulls(h):
         a hash without nulls
     """
     if isinstance(h, list):
-        return map(delete_nulls, h)
+        return [delete_nulls(i) for i in h if i is not None]
```

Comment:
What you're doing here is changing the behavior. If you want the code to do the same as in Python 2, you need to do:
```suggestion
        return [delete_nulls(i) for i in h]
```

Reply:
Yes that's true, just realized it, thanks for your input.